pkg/kn/traffic/compute_test.go

```diff
@@ -265,6 +266,18 @@ func TestComputeErrMsg(t *testing.T) {
 			[]string{"--traffic", "@latest=19,echo-v1=71"},
 			"given traffic percents sum to 90, want 100",
 		},
+		{
+			"verify error for values out of range given to percent",
+			append(newServiceTraffic([]v1alpha1.TrafficTarget{}), newTarget("", "", 100, true)),
+			[]string{"--traffic", "@latest=-100"},
+			"invalid value for traffic percent -100, expected 0 <= percent <= 100",
+		},
+		{
+			"repeatedly spliting traffic to the same revision",
+			append(newServiceTraffic([]v1alpha1.TrafficTarget{}), newTarget("", "", 100, true)),
+			[]string{"--traffic", "echo-v1=40", "--traffic", "echo-v1=60"},
+			"repetition of identifier echo-v1 is not allowed, use only once with --traffic flag",
```

Comment:
We should make this error message read clear for user, for example here echo-v1 is a revision, it could be a tag as well or even @latest.

All 3 are revision references, so if we instead say
```
repetition of revision reference echo-v1 is not allowed, use only once with --traffic flag
```

Reply:
We can actually filter this check for `@latest` and say

```
repetition of identifier @latest is not allowed, use only once with --traffic flag
```

and keep the message same for tag and revision as mentioned in above comment

Reply:
Thanks for the suggestion! Will update it.